1706_CLT_3 Fixes

Crank1986 opened the pull request:
Обновлены схемы.
Исправлены ошибки в названиях и связях.

Crank1986 committed:
Final Release.

Обновлены схемы.
Исправлены ошибки в названиях и связях.

sov-87 commented on DB/DATA/Hospital_City.sql:
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `Hospital_City` (
```
клиника может быть одновременно в нескольких городах? здесь нет ошибки?
или здесь подразумевается какая-то сеть клиник, а не конкретный экземпляр клиники?

Crank1986 committed:
Исправления типа данных:

- Varchar2 заменён на Varchar() - где в скобках указывается конкретное количество символов, необходимое для поля.

Исправление полей и таблиц:
- Добавлены поля отдельно для имени, фамилии и отчества докторов и пациентов;
- Нейминг таблиц приведён к нотации наименования в единственном числе;
- Добавлен постфикс FK для полей, являющихся внешними ключами;
- Добавлен ИНН пациентов - как бизнес ключ для данной таблицы;
- Добавлены контакты пациентов - электронная почта и телефон;
- Для таблицы Drugs добавлен Артикул, глобальное и локальное наименования, активное действующее вещество;
- Таблица Drugs связана с таблицей Dicease;
- В таблицу Clinical_trial добавлены поля начала, окончания исследования, а также наименование исследования;
- Жалобы пациентов выведены в отдельную таблицу Appeals, которая связана с таблицей Patient.
- Удалены таблицы City и Hospital_city, вместо них в таблицу Hospital добавлено поле содержащее адрес.
- Для таблицы Hospital Добавлены поля Patients и Doctors, в которых указаны привязанные к клинике пациенты и доктора, также добавлено поле Specialization - которое имеет связь с таблицей Specialization и обозначает направление деятельности клиники.
-

Crank1986 committed:
Исправлен регистр наименований полей и таблиц

sov-87 commented on DB/DATA/Hospital.sql:
```diff
@@ -0,0 +1,11 @@
+  "hospital_name" varchar(150),
+  "address" varchar(200),
+  "patients_fk" number,
+  "doctors_fk" number,
```
у госпиталя предполагается ровно один доктор?

sov-87 commented on DB/DATA/Clinical_trial.sql:
```diff
@@ -0,0 +1,22 @@
+  "start_date" datetime,
+  "end_date" datetime,
+  "is_placebo" bool,
+  "patient_inn_fk" number,
```
предполагается, что в испытании участвует ровно один клиент, ровно одна клиника, ровно один доктор?

sov-87 commented on DB/DATA/Dicease.sql:
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE "dicease" (
+  "id" number PRIMARY KEY,
+  "dicease_name" number,
```
для колонки с наименованием добавление имени таблицы лишнее. во всех таблицах.

Crank1986 committed:
Исправлены связи между таблицами Hospital, Clinical_trial, Patient, Doctor.

Исправлены наименования полей для таблиц.

Crank1986 committed:
Добавлены связи для Patient и Hospital, Doctor и Hospital.

Crank1986 committed:
Добавлено описание связей для каждой таблицы.

Crank1986 committed:
Исправлены связи таблицы.

Исправлена схема.
Добавлено поле "Result" в таблицу Clinical_Trials
